src/utils/octokit.ts

```diff
@@ -21,11 +12,14 @@ export async function getOctokit(): Promise<GitHub> {
   octokit =
     octokit ||
     new GitHub({
-      auth: (
-        (await getAuthProvider()({
-          type: 'installation',
-        })) as InstallationAccessTokenAuthentication
-      ).token,
+      authStrategy: createAppAuth,
+      auth: {
+        appId: process.env.APP_ID,
+        privateKey: process.env.PRIVATE_KEY,
+        installationId: process.env.INSTALLATION_ID,
+        clientId: process.env.CLIENT_ID,
+        clientSecret: process.env.CLIENT_SECRET,
+      },
```

Comment:
I think previously we were just passing in the token here, will this method take the entire object and still work?

Reply:
Should! This is one of the forms shown in the docs: https://github.com/octokit/auth-app.js/?tab=readme-ov-file#usage-with-octokit